deepchem/models/keras_model.py

```diff
@@ -936,10 +956,14 @@ def restore(self, checkpoint=None):
       the path to the checkpoint file to load.  If this is None, the most recent
       checkpoint will be chosen automatically.  Call get_checkpoints() to get a
       list of all available checkpoints.
+    restore_from: str, default None
```

Comment:
It still has the old argument name here.

Reply:
Fixed.